http-client/Network/HTTP/Client/Types.hs

```diff
@@ -665,7 +665,7 @@ instance Show Request where
         , "  requestHeaders       = " ++ show (DL.map (redactSensitiveHeader $ redactHeaders x) (requestHeaders x))
         , "  path                 = " ++ show (path x)
         , "  queryString          = " ++ show (queryString x)
-        --, "  requestBody          = " ++ show (requestBody x)
+        , "  requestBody          = " ++ show (requestBody x)
```

Comment:
`RequestBody` does not have a `Show` instance, so I don't think this is possible, at least not in the general case.